as-pect.config.js

```diff
@@ -15,7 +16,7 @@ export default {
   },
 
   // Enable code coverage.
-  coverage: ['assembly/**/*.ts'],
+  // coverage: ['assembly/**/*.ts'],
```

Comment:
please remove commented out lines.

Why don't you enable coverage in local environment?

Reply:
I removed the comment, thanks!